src/main/java/fi/aalto/cs/apluscourses/ui/ideactivities/PopupTransparencyHandler.java

```diff
@@ -0,0 +1,65 @@
+package fi.aalto.cs.apluscourses.ui.ideactivities;
+
+public class PopupTransparencyHandler {
+
+  private final TransparentComponent component;
+
+  private long lastTransitionUpdate;
+
+  private static final float MIN_TRANSPARENCY = 0.55f;
+  private static final float MAX_TRANSPARENCY = 1.0f;
+  private static final float TRANSITION_STEP = 0.0015f;
+
```

Comment:
(1.0 - 0.55) / 0.0015 = 300. So does that mean the animation takes 0.3 seconds? Just trying to understand :)

Reply:
Yes